static/js/extension.js

```diff
@@ -32,6 +32,18 @@ class Extension {
     return Menu.addExtensionItem(this, name);
   }
 
+  /**
+   * Load your html page here
+   *
+   * @returns {Promise} Promise which should resolve as soon as the extension
+   *                    is ready to be shown
+   */
+  load() {
+    return new Promise((resolve) => {
```

Comment:
`return Promise.resolve()` would be simpler.

Reply:
Done